src/main/java/org/mastodon/views/trackscheme/display/TrackSchemeFrame.java

```diff
@@ -79,7 +82,9 @@ public TrackSchemeFrame(
 				graph,
 				highlight,
 				focus,
-				timepoint,
+				timepointModel,
+				fadingTimepointModel,
+				fadedModel,
```

Comment:
The fadingTimepointModel isn't required here. Please add listeners to the fadedModel instead.

Reply:
Thanks for the advice. I had this idea as well but did not find a solution before.
Now, I tried again. It is solved now via listeners() and the fadingTimepointModel is not needed here anymore. However, I am not sure, if my implementation meets your intention. Please check.

Reply:
After this discussing this again I introduced a FadingListener that is implemented by the TrackSchemePanel.